### PR TITLE
Ft/User Feedback Message Page

### PR DESCRIPTION
**Pull request by nikunjsdev**
https://github.com/adrienshen/saleor-storefront/projects/1#card-39392243

### CODE REVIEW

File: src/components/UserActionFeedback/UserActionList.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import cartIcon from "images/cart-empty.svg";
+
+export const UserAction = {
```
Comment (zshanhui) [inline src/components/UserActionFeedback/UserActionList.ts]: Rename to PageFeedbackTypes
and create a typescript interface to type this object.

File: src/components/UserActionFeedback/stories.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { UserActionFeedback } from ".";
+
+const DEFAULT_PROPS = {
+  page: "message_sent",
```
Comment (zshanhui) [inline src/components/UserActionFeedback/stories.tsx]: "message_sent" should be defined in an `enum`

**Review by zshanhui — CHANGES_REQUESTED** [review]

File: src/components/UserActionFeedback/UserActionList.ts (hunk 1 above)
Comment (nikunjsdev) [reply src/components/UserActionFeedback/UserActionList.ts]: Done

File: src/components/UserActionFeedback/stories.tsx (hunk 2 above)
Comment (nikunjsdev) [reply src/components/UserActionFeedback/stories.tsx]: Done

**Review by zshanhui — APPROVED** [review]
lgtm!

**Pull request merged**